src/server/server.js

```diff
@@ -394,4 +404,58 @@ function processPluginHtml(html, pluginId) {
     });
 }
 
+function localizeHtmlFile(htmlFile, html, lang) {
+    // Replace localized strings
+    var targetJsonFile = getI18nJsonFile(htmlFile, lang);
+    if (utils.existsSync(targetJsonFile)) {
+        try {
+            var locItems = require(targetJsonFile);
+        } catch (e) {
+            // Don't care if it fails
+            return;
+        }
+
+        locItems.forEach(function (locItem) {
+            html = replaceLocalizationItem(html, locItem);
+        });
+    }
+    return html;
+}
+
+function replaceLocalizationItem(html, locItem) {
+    // To try and be as targeted as possible, we replace attribute by looking for 'attribute="value"', and
+    // regular text by looking for '>text<'. If we don't find it that way, we revert to a simpler approach.
+
+    var attribute = locItem.attribute;
+    var source, target;
+    if (attribute) {
+        source = attribute + '="' + locItem.source + '"';
+        target = attribute + '="' + locItem.target + '"';
+    } else {
+        source = '>' + locItem.source + '<';
+        target = '>' + locItem.target + '<';
+    }
+    var newHtml = html.replace(source, target);
```

Comment:
These replacements will only replace the first instance. Do we have any localized strings which appear multiple times?